packaging/linux/smoketest/README.md

```diff
@@ -0,0 +1,9 @@
+# linux smoketests
```

Comment:
The word `smoketest` has so many different meanings that it might be good to add some words about what it means here in particular.